src/chigame/api/tests/factories.py

```diff
@@ -106,3 +106,18 @@ class Meta:
         model = Chat
 
     tournament = SubFactory(TournamentFactory)
+
+
+class LobbyFactory(DjangoModelFactory):
+    class Meta:
+        model = Lobby
+
+    match_status = Lobby.Lobbied
+    name = Faker("word")
+    game = SubFactory(GameFactory)
+    game_mod_status = Lobby.Default_game
```

Comment:
This should also be allowed to be one of the two game options (default / unique)

```py
class Lobby(models.Model):
    """
    A lobby that users can join before starting a match.
    The match can start as soon as min_players join the lobby.
    """

    Lobbied = 1
    Viewable = 2
    Finished = 3
    STATUS = ((Lobbied, "Lobbied"), (Viewable, "In-Progress"), (Finished, "Finished"))

    Default_game = 1
    Modified_game = 2
    MODS = ((Default_game, "Default Game"), (Modified_game, "Modified Game"))

    match_status = models.PositiveSmallIntegerField(choices=STATUS, default=1)
```